### PR TITLE
Update README.md of github organization page

### PR DESCRIPTION
**Pull request by Sakrecoer**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,11 +1,14 @@
-# 👋 Welcome
+# 🕊️ Free to share code 👩🏽‍💻 Code to share freedom
 
-Dyne.org is a non-profit **free software foundry** and a merry **hacker community**.
+🖖 Welcome, fellow dyne! 
 
-We have more than **20 years of expertise** putting computers at work to **empower people**.
+[Dyne.org](https://dyne.org) is a non-profit **free software foundry** and a merry **hacker community**.<br>We have nearly **30 years of expertise** putting computers at work to **empower people**.
```
Comment (puria) [inline profile/README.md]: avoid the `<br/>` if possibile you can achive that with two newlines in markdown

File: profile/README.md [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
 
-We are **free to share code** and we **code to share freedom** ✊🏽
+**💎 To learn more, head over to [our website](https://dyne.org)**
 
-## TL;DR head over [our website](https://dyne.org) (links to many chat channels)
+**🌐 Engage with us on the [Social Networks](https://dyne.org/linktree)**
+
+**📢 Get news from us on [our blog](https://news.dyne.org) and tune in to our corner of the [fediverse](https://fed.dyne.org)**
+
+[![Github organization banner](https://github.com/Sakrecoer/.github-1/assets/3484593/7ada54a5-95e3-4f3d-8bb3-17101790a4ae)](https://github.com/sponsors/dyne)
```
Comment (puria) [inline profile/README.md]: mayb you want to add the asset directly in this repo in place of the `Sakrecoer/.github-1`

File: profile/README.md (hunk 1 above)
Comment (jaromil) [reply profile/README.md]: wells potted, seems good also to keep on same line

File: profile/README.md (hunk 2 above)
Comment (Sakrecoer) [reply profile/README.md]: yes! i didn't realize, thank you. Problem is i don't have access to the https://github.com/dyne/.github

File: profile/README.md (hunk 1 above)
Comment (Sakrecoer) [reply profile/README.md]: 2 news lines creates 2 new lines :/ (why, mr Markdown? Please explain yourself!)  
Github markdown uses `<br>` wildly and widely. But sure, 2 lines it is!

**Pull request merged**